controllers/usersController.js

```diff
@@ -78,17 +78,17 @@ exports.register = async (req, res) => {
 		name: req.body.name,
 		familyName: req.body.familyName,
 		displayName: req.body.displayName,
-		password: req.body.password,
+    password: req.body.password,
+    pushNotificationsAllowed: !!req.body.pushNotificationsAllowed,
```

Comment:
```
const obj = {
  value: 5,
  node: null
};
!!obj.value; // true
!!obj.node;  // false
```
Basically checks if its there or not in the request if it is returns true if not returns false.. otherwise would have to write
```
pushNotificationsAllowed: req.body.pushNotifications ? true : false
```

Reply:
Clever.